python/foglamp/plugins/north/omf/omf.py

```diff
@@ -131,6 +133,11 @@
         "description": "JQ formatted filter to apply (only applicable if applyFilter is True)",
         "type": "string",
         "default": ".[]"
+    },
+    "destination_type": {
+        "description": "Destination: 1-OMF, 2-Elasticsearch, 3-OCS",
+        "type": "integer",
+        "default": "1"
```

Comment:
why do we need this if its OMF and we always it to be 1, same for others

Reply:
Because, sending_process.py picks/creates destination_id from destination_type (and stream_id from destination_id) from plugin's configuration. Hence it is required to define destination_type in plugin's configuration.

Reply:
This is not required. The plugin is an OMF plugin, it does not need to be told it is OMF, and it can never be OCS or Elasticsearch